tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -155,20 +172,46 @@ def test_createami_wrong_pcluster_version(
     wrong_ami = pcluster_ami_without_standard_naming(wrong_version)
     # Networking
     vpc_id = vpc_stack.cfn_outputs["VpcId"]
-    subnet_id = vpc_stack.cfn_outputs["PublicSubnetId"]
-    networking_args = f"--vpc-id {vpc_id} --subnet-id {subnet_id}"
+    networking_args = ["--vpc-id", vpc_id, "--subnet-id", vpc_stack.cfn_outputs["PublicSubnetId"]]
 
     # Custom Cookbook
     custom_cookbook = request.config.getoption("createami_custom_chef_cookbook")
-    custom_cookbook_args = "" if not custom_cookbook else f"-cc {custom_cookbook}"
+    custom_cookbook_args = [] if not custom_cookbook else ["-cc", custom_cookbook]
 
-    command = (
-        f"pcluster createami -ai {wrong_ami} -os {os} -r {region} -c {cluster_config.as_posix()} "
-        f"{custom_cookbook_args} -i {instance} {networking_args}"
+    pcluster_createami_result = run_command(
```

Comment:
The code is very similar to the one in the previous test.
What about creating an utility function to compose the createami command or just have a single test with some parameters to test wrong os and wrong ami failures?